libs/payments/legacy/src/lib/utils/buildSp2RedirectUrl.spec.ts

```diff
@@ -37,4 +37,17 @@ describe('buildSp2RedirectUrl', () => {
       'http://content-server.com/subscriptions/products/prod_123?plan=price_123&flow_id=one&flow_begin_time=123'
     );
   });
+
+  it('should send no query params', () => {
```

Comment:
?plan=price_123 is a query param. Can we rename this to be "should send no query params besides price"?